resources/cmake/UseOpenSSL.cmake

```diff
@@ -0,0 +1,1104 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
```

Comment:
Since this file is from our internal infrastructure, do not change the copyright header

Reply:
Changed back to 2018.